FilecoinFoundationWeb/filecoin-foundation
# Command to generate page template files

## Note

Command `npm run generate:page -- whatever-page-name ` generates:
- new folder `whatever-page-name` under `app/src `with the following files:` page.tsx`, `layout.tsx`, `utils/generateStructuredData.ts`
- it creates `whatever-page-name.md` under `content/pages`
- it also updates `paths.ts` with necessary changes when adding a page

## How to test
- run `npm run generate:page -- page-name` and check generated files
- run `http://localhost:3000/page-name` and check if the page has been created
